build/export_generators/hardcoded-cmake/disclaimer.jinja

```diff
@@ -1,13 +1,6 @@
-# This file was generated by the build system used internally in Yandex and called "ya"
-# (https://github.com/yandex/yatool).
+# This file was generated by the YaTool build system (https://github.com/yandex/yatool), 
+# from a source YaTool build configuration provided in ya.make files.
 #
-# Configuration files of ya build system are usually named ya.make. If ya.make file is present
-# at the root of the repository, then this repository supports ya build.
+# Please refer to the build instructions in the repository to see policies for 
+# manual changes in this file.
 #
-# If the repository supports both CMake and ya build configurations, please modify both of them.
```

Comment:
This part of the disclamer is important for some projects and should should not be removed.